packages/libs/eda/src/lib/map/analysis/mapTypes/plugins/BubbleMarkerMapType.tsx

```diff
@@ -786,5 +786,5 @@ function useMarkerData(props: DataProps) {
         boundsZoomLevel,
       },
     };
-  }, [rawMarkersResult, legendDataResult.data, overlayConfig]);
+  }, [rawMarkersResult, legendDataResult.data, overlayConfig, boundsZoomLevel]);
```

Comment:
Good catch!

Reply:
Thanks @bobular! 👍 So, you don't want to show snackbar if the side panel is already open and does not point out marker configuration panel (Donuts, Bar plots, or Bubbles), right? This is updated~

Reply:
Thank you for your confirmation, @bobular  👍 I will merge this PR, finally 🎉